core/deployment/src/main/java/io/quarkus/deployment/builditem/NativeImageEnableAllTimeZonesBuildItem.java

```diff
@@ -2,6 +2,14 @@
 
 import io.quarkus.builder.item.MultiBuildItem;
 
+/**
+ * All timezones are now included in native executables by default so this build item does not change the build behavior
+ * anymore.
+ * <p>
+ * Keeping it around for now as it marks the extension requiring all the timezones
+ * and better wait for the situation to fully settle on the GraalVM side
+ * before removing it entirely.
+ */
 public final class NativeImageEnableAllTimeZonesBuildItem extends MultiBuildItem {
```

Comment:
Maybe mark it with `@Deprecated`?

Reply:
So I thought about that and decided not to for now. Otherwise people would remove the marker and I think keeping it for a few versions to be sure they don't retropedal is a good thing.

Reply:
OK, in the end I made it deprecated.

Reply:
Cool